cloud/amazon/acm_certificate_facts.py

```diff
@@ -0,0 +1,504 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: acm_certificate_facts
+short_description: Retrieve the facts of a Certificate from AWS Certificate Manager.
+description:
+  - Retrieve the attributes of a Certificate from AWS Certificate Manager.
+version_added: "2.2"
+author: "Allen Sanabria (@linuxdynasty)"
+requirements: [boto3, botocore]
+options:
+  name:
```

Comment:
`name` is mentioned here, but examples and the rest of the code call it `domain_name`.